src/styles/main.scss

```diff
@@ -1,3 +1,23 @@
-body {
+@import "./utils/variables";
+@import "./utils/functions";
+@import "./blocks/header.scss";
+@import "./blocks/nav.scss";
+@import "./blocks/card.scss";
+@import "./blocks/stars.scss";
+@import "./blocks/main.scss";
+@import "./blocks/catalog.scss";
```

Comment:
its better to import blocks in their order in html file, in your case it should be main > catalog > card > stars, its will prevent wrong order of styles uploading